starfish/core/types/_spot_finding_resutls.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Mapping, MutableMapping, Optional, Tuple
+
+from starfish.core.types import Axes, SpotAttributes
+
+AXES_ORDER = (Axes.ROUND, Axes.CH)
+
+
+class SpotFindingResults:
```

Comment:
Given this wraps a dict, I would prefer this be implemented to mimic the dictionary container, and more clearly document what it contains. This should simplify a lot of the accessors you've written. For example, 

```diff
+     def values(self):
-     def all_spots(self):
-         """
-         Return all SpotAttributes across rounds and chs.
-         """
          return self._results.values()
```

See: https://docs.python.org/2.5/ref/sequence-types.html

Reply:
Yea so I started down this route but found that things became too confusing. For example get_spots_for_round_ch would instead just be  __getitem__(key) and then when you used it it was totally clear what you were getting. I'm not super opinionated on this though so open to changing it back if that's what you'd prefer I just thought this was more descriptive